<!-- FroboScout launch file -->

<launch>
	<node pkg="hmi_support_frobit" name="hmi_support_frobit_node" type="hmi_support_frobit_node.py" output="screen">
		<param name="automode_pub" value="/fmPlan/automode" />
		<param name="deadman_pub" value="/fmSafe/deadman" />
		<param name="cmd_vel_pub" value="/fmCommand/cmd_vel" />
		<param name="ui_str_control_sub" value="/ui_str_control" />
		<!--<param name="max_linear_velocity" value="50.7"/>
        <param name="max_angular_velocity" value="50.5"/>-->
        <param name="max_linear_velocity" value="1.0"/>
        <param name="max_angular_velocity" value="0.8"/>
	</node>
</launch>

